### ISSUE TITLE
Please make diff colors configurable

### ISSUE THREAD

**Issue by DanielGibson** [prati0100/git-gui#86]
Being red-green colorblind, git gui's diff view is hard to parse for me.
Right now I'm patching git gui to replace `#00a000` with `blue`, but each time git gets updated, I have to reapply the patch..

It would be nice if the colors could just be an configurable in the options menu, like in gitk.  
Or maybe the `color.diff.*` settings of git config could be used.

Either way, it would be very helpful for colorblind people (that in many countries make up 8-10% of the male population!), if the diff colors would be configurable

**Comment by daira** [comment]
I intend to make these configurable in daira/git-gui#1. It's required anyway for contrast adjustment in dark mode, but thanks for reminding me of the colour blindness motivation.
